packages/sdk/docs/interfaces/MessageReceipt.md

```diff
@@ -0,0 +1,32 @@
+[@eth-optimism/sdk](../README.md) / [Exports](../modules.md) / MessageReceipt
+
+# Interface: MessageReceipt
+
+CrossDomainMessage receipt.
+
+## Table of contents
+
+### Properties
+
+- [receiptStatus](MessageReceipt.md#receiptstatus)
+- [transactionReceipt](MessageReceipt.md#transactionreceipt)
+
+## Properties
+
+### receiptStatus
+
+• **receiptStatus**: [`MessageReceiptStatus`](../enums/MessageReceiptStatus.md)
+
+#### Defined in
+
+[packages/sdk/src/interfaces/types.ts:200](https://github.com/ethereum-optimism/optimism/blob/4e370ce3/packages/sdk/src/interfaces/types.ts#L200)
```

Comment:
Why are some types from `/packages/sdk/src/interfaces/types.ts`  documented in `docs/interfaces/` and some in `docs/enums`? e.g. `packages/sdk/docs/enums/MessageDirection.md` vs `packages/sdk/docs/interfaces/MessageReceipt.md`